sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/AzureMonitorOptions.cs

```diff
@@ -60,5 +66,16 @@ internal void SetValueToExporterOptions(AzureMonitorExporterOptions exporterOpti
                 exporterOptions.Transport = Transport;
             }
         }
+
+        internal void SetValueToLiveMetricsExporterOptions(LiveMetricsExporterOptions liveMetricsExporterOptions)
+        {
+            liveMetricsExporterOptions.ConnectionString = ConnectionString;
```

Comment:
just to confirm - this will always be populated from the overall setting, and there is no way the customer can override the connectionstring for Livemetrics alone. (in other words, will it ever be diff from the connectionstring used by exporter?)

Reply:
The ConnectionString contains two different endpoints, one for ingestion and one for livemetrics.

Reply:
This is correct, customer won't be able override only for Livemetrics.